### PR TITLE
Remove LD_LIBRARY_PATH

### PR DESCRIPTION
**Pull request by saraedum**

### CODE REVIEW

**Comment by videlec** [comment]
This is also needed for `make distcheck` appearing just after.

**Comment by saraedum** [comment]
Hm…I ran this locally in a trusty docker container and it worked. For some reason, on TravisCI it's building as
```
libtool: link: g++ -std=gnu++11 -g -O2 -o renfxx/test/.libs/t-stream renfxx/test/t-stream.o  ./.libs/libeanticxx.so ./.libs/libeantic.so -lgmpxx -larb -lflint -lgmp -fopenmp -Wl,-rpath -Wl,/home/travis/build/videlec/e-antic/e-antic-0.1.2/_inst/lib
```

whereas on my local docker container it's using:
```
libtool: link: g++ -std=gnu++11 -g -O2 -o renfxx/test/.libs/t-stream renfxx/test/t-stream.o  ./.libs/libeanticxx.so ./.libs/libeantic.so -lgmpxx -larb -lflint -lgmp -fopenmp
```

**Comment by saraedum** [comment]
It's odd that `make check` does not print the failed logs. The logs only show if the runner is set to "serial" which can only be changed globally in `Makefile.am`.

File: .travis.yml [hunk 1]
```diff
@@ -1,43 +1,36 @@
...
   - tar xf 2.16.0.tar.gz
   - cd arb-2.16.0
-  - ./configure --prefix=${PREFIX} --with-flint=${PREFIX} && make && sudo make install
+  - ./configure --prefix=/usr && make && sudo make install
   - cd ..
-  # check linking of installation
-  - ldd ${PREFIX}/lib/libflint.so
-  - ldd ${PREFIX}/lib/libarb.so
-  # build e-antic
+  # build E-ANTIC
```
Comment (saraedum) [inline .travis.yml]: That's always a bit confusing to me. I think it's `ANTIC` officially, right? So should it also be `E-ANTIC` everywhere?

Comment (videlec) [reply .travis.yml]: No, not everywhere. On the [README](https://github.com/wbhart/antic) it is ANTIC in the title and then Antic.

**Comment by saraedum** [comment]
What's the status here? Is there anything left that I need to do?

File: .travis.yml [hunk 2]
```diff
@@ -1,43 +1,36 @@
...
-  # build flint
-  - wget http://flintlib.org/flint-2.5.2.tar.gz
-  - tar xf flint-2.5.2.tar.gz
-  - cd flint-2.5.2
-  - ./configure --prefix=${PREFIX} && make && sudo make install
-  - cd ..
-  # build arb
+  # install FLINT (currently, there is no need to build as we are using a standard version)
+  - sudo apt-get install libflint-dev
+  # - wget http://flintlib.org/flint-2.5.2.tar.gz
```
Comment (videlec) [inline .travis.yml]: can you get rid of these commented lines?

Comment (saraedum) [reply .travis.yml]: Sure. They can go away. I thought you might have to go back to a development version of Flint at some point.

Comment (videlec) [reply .travis.yml]: I did it myself :-)

Comment (videlec) [reply .travis.yml]: Actually we should ideally be testing both: the version from ubuntu and the development version.

**Pull request merged**